libs/execution_context/ebpf_program.c

```diff
@@ -56,7 +56,9 @@ typedef struct _ebpf_program
     const void* info_extension_provider_binding_context;
     const ebpf_extension_data_t* info_extension_provider_data;
     // Program type specific helper function count.
-    uint32_t provider_helper_function_count;
+    uint32_t program_specific_helper_function_count;
```

Comment:
shouldn't be program-type specific, rather than program specific?

Reply:
yes ideally it should be `program_type_specific`, but the names were becoming very long (here and other places), so I was looking for ways to possibly shorten it. I can change it to `program_type_specific` if this looks confusing.

Reply:
Updated to `program_type_specific_ ...`